README.md

```diff
@@ -1,234 +1,165 @@
+A build system for [Mapbox GL](https://docs.mapbox.com/mapbox-gl-js/style-spec/) and [Maplibre GL](https://maplibre.org/maplibre-style-spec) styles that enables modular, maintainable map styling through layer composition and template-based development.
 
-A build system lets you more easily maintain stylesheet variations by removing the need to make duplicative changes across multiple stylesheets.
+## What This Tool Does
 
-## Usage
-
-### Style definition
+Instead of maintaining massive JSON stylesheets, `map-gl-style-build` lets you:
 
-This script assumes you have two directories:
+- **Break stylesheets into manageable layer files** - Each map layer becomes its own JavaScript module
+- **Create style variants easily** - Define multiple styles by composing layers with different contexts
+- **Work iteratively** - Make changes to individual layers and rebuild instantly
+- **Maintain consistency** - Share common styling logic across multiple map styles
+- **Version control effectively** - Track changes at the layer level, not the entire stylesheet
 
-1.  The **styles** directory, which contains each style you want to build. Each style is defined as a JS module that exports two plain JS objects:
-    1. `context`: The variables this style defines that will be passed to layers during the build
-    2. `template`: The style, which is a [MapLibre GL style](https://maplibre.org/maplibre-style-spec/) or [Mapbox GL style](https://docs.mapbox.com/mapbox-gl-js/style-spec/), the only difference being that `layers` is an array of layer ids.
-2.  The **layers** directory, which contains each layer that will be included in a style. Each layer is defined as a JS module that exports a default function. The function takes one parameter: `context`, which contains the variables passed from the style. The function
-    must return two objects:
-    1. `baseStyle`: The base style object
-    2. `overrides`: Any overrides for to the `baseStyle`, may be an empty object if no overrides are necessary
+## Who is this tool for?
 
-See the `examples` directory for examples.
+- Users who need to maintain multiple stylesheets that are variations on one another and contain redundant style definitions
+- Users who need to maintain a single stylesheet and prefer to work on layers in a more composable and maintainable way
 
-### Using the script
+This library is a compiler that turns your JS files into renderable JSON stylesheets. To use it effectively, it expects you to set up a specific directory and file structure. More on this below.
 
-Once installed using your package manager:
+## Installation
 
 ```bash
-map-gl-style-build
-    --style-dir=templates/styles
-    --layer-dir=templates/layers
-    --out-dir=build
+# Install the tool in your map project directory
+yarn add map-gl-style-build@https://github.com/stamen/map-gl-style-build
```

Comment:
Not too strongly opinionated, but do we want to be using `https` as the example over `ssh`?

Reply:
I have no opinion about this at all. @ebrelsford?